frontend/components/Forms/Files/Files.js

```diff
@@ -0,0 +1,82 @@
+import { ErrorMessage, Field, Form, Formik } from "formik";
+import React, { useContext } from "react";
+import { FormContext } from "../../Overlay/CreateProject.js";
+import * as yup from "yup";
+import classes from "../../../styles/Forms.module.css";
+
+function Files() {
+  const { activeStepIndex, setActiveStepIndex, formData, setFormData } =
+    useContext(FormContext);
+
+  const renderError = (message) => (
+    <p className={classes.errorMessage}>{message}</p>
+  );
+
+  const ValidationSchema = yup.object().shape({
+    files: yup.mixed().required("OTFs or TTFs are required"),
+    projectName: yup.string(),
+    description: yup.string(),
+    /* .test(
+        "format",
+        "Only the following formats are accepted: .ttf or .otf",
+        (format) =>
+          SUPPORTED_FORMATS.includes(["application/x-font-ttf", "font/*"])
+      ) */
+  });
+
+  return (
+    <Formik
+      initialValues={{
+        files: "",
+        projectName: "",
+        description: "",
+      }}
+      validationSchema={ValidationSchema}
+      onSubmit={(values) => {
+        const data = { ...formData, ...values };
+        setFormData(data);
+        setActiveStepIndex(activeStepIndex + 1);
+      }}
+    >
+      <Form className={classes.formContainer}>
+        <div className={classes.title}>Upload OTF and/or TTF files!</div>
+        <div className={classes.formContainer}>
+          <Field
+            type="file"
```

Comment:
Not sure Formik Field component allows it but regular file input html element allow passing in these two attributes...

`accept`   - Set which file types are allowed to be selected ( https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#limiting_accepted_file_types )

`multiple` - Set if multiple files are allowed to be selected ( https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/multiple )

```
<input accept=".ttf,.otf,.woff,.woff2" type="file" name="fonts" multiple="multiple" />
```